app/src/main/java/bubbleheads/buy_cook/CategoriesAdapter.java

```diff
@@ -49,6 +43,15 @@ public MyViewHolder onCreateViewHolder(final ViewGroup parent, final int viewTyp
     public void onBindViewHolder(final MyViewHolder holder, final int position) {
         final TextView categoryName = (TextView) holder.itemView.findViewById(R.id.categories_name);
```

Comment:
you already have this view as field in viewHolder. Why do you take it again, just reuse the field